packages/frontend/editor-ui/src/features/mcpAccess/SettingsMCPView.vue

```diff
@@ -83,6 +83,11 @@ const tableActions = ref<Array<UserAction<WorkflowListItem>>>([
 	},
 ]);
 
+const isOwner = computed(() => usersStore.isInstanceOwner);
```

Comment:
We also have `hasPermission(['instanceOwner']` helpers from `@/utils/rbac/permissions`. There's also a helper for doing rbac queries if endpoints are using the RBAC system, and with that you could probably do something like

```
const canToggleMCP = computed(() => hasPermission(['rbac'], { rbac: { scope: 'mcp:manage' } }));`